README.markdown

```diff
@@ -311,6 +311,27 @@ else {
 ```
 
 * There should be exactly one blank line between methods to aid in visual clarity and organization. Whitespace within methods should separate functionality, but having too many sections in a method often means you should refactor into several methods.
+ 
+* No blank lines between the last method and closing bracket of a class/struct/enum.
+
+**Preferred:**
+```swift
+class SomeClass {
+  func someFunc() {
```

Comment:
github's editor is weird, here's how it looks while editing:
<img width="164" alt="screen shot 2016-08-22 at 3 19 32 pm" src="https://cloud.githubusercontent.com/assets/16186741/17854515/f35af19e-687b-11e6-8c9c-afaf8fa8c36c.png">